src/perf-tests/mock-scripts/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3
```

Comment:
Overall, the PR is fine, I just dislike this file.

Reply:
Clarification: There is too much hard coded info – consider using args for the build instead.